backend/src/migrations/1543523539612_init.js

```diff
@@ -0,0 +1,98 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.createTable('users', {
+        email: { type: 'varchar(254)', primaryKey: true},
+        first_name: {type: 'varchar(30)'},
+        last_name: {type: 'varchar(150)'},
+        password: {type: 'varchar(128)'},
+        bio: {type: 'varchar(500)'}
+    });
+
+/*****************************************************************************/
+    pgm.createTable('activities', {
+        id: 'id',
```

Comment:
What does 'id' mean? Something like "Primary key int auto_increment not_null"?

Reply:
It's a shortcut in `pg` library, basically its a type alias for numerical primary key.